src/libsystemd/sd-journal/journal-verify.c

```diff
@@ -1316,6 +1349,15 @@ int journal_file_verify(
         if (entry_array_fd >= 0)
                 safe_close(entry_array_fd);
 
+        if (data_fp)
+                fclose(data_fp);
+
+        if (entry_fp)
+                fclose(entry_fp);
+
+        if (entry_array_fp)
+                fclose(entry_array_fp);
```

Comment:
replace by `_cleanup_fclose_` use, see above

Reply:
(actually you get extra points if you replace the safe_close() calls for the underlying fds with _cleanup_close_() too, in a similar fashion

Reply:
done